test/support/helpers/platform.js

```diff
@@ -43,24 +42,23 @@ function onMacOSAtLeast (minRelease /*: MacOSReleaseInfo */, spec /*: Function *
 //     })
 //
 function onMacOSAtMost (maxRelease /*: MacOSReleaseInfo */, spec /*: Function */) {
-  const describe = macOSRelease.isAtMost(maxRelease)
-    ? mocha.describe
-    : mocha.describe.skip
+  const describeOrSkip = macOSRelease.isAtMost(maxRelease)
+    ? describe
+    : describe.skip
 
-  describe(`on ${macOSRelease.name(maxRelease)} or lower`, spec)
+  describeOrSkip(`on ${macOSRelease.name(maxRelease)} or lower`, spec)
 }
 
-// $FlowFixMe
-function onPlatforms (...platformsAndSpec) {
-  const spec = platformsAndSpec.pop()
-  const expectedPlatforms = platformsAndSpec
+function onPlatforms (...platformsAndSpec /*: any[] */) {
```

Comment:
I had to cast the parameters array with `any` as it can be a `string[]::Function`.
We use it like this `onPlatforms("win32", "linux", callback)` or `onPlatforms("linux", callback)`. So it may be refactored to express a little better what it is all about. For instance it could take an array of platforms as `string` and a callback parameter as a `Function` instead of `any`.